components/taxonomy/.gitignore

```diff
@@ -0,0 +1,2 @@
+target/
```

Comment:
Good catch. That particular file is not necessary as `.gitignore` at the root of the repo already takes care of `target/` and `Cargo.lock`. I added an extra commit to clean it up